packages/catalog-process/src/config/config.ts

```diff
@@ -16,8 +16,18 @@ const CatalogProcessConfig = CommonHTTPServiceConfig.and(ReadModelDbConfig)
       .object({
         ESERVICE_DOCUMENTS_PATH: z.string(),
         PRODUCER_ALLOWED_ORIGINS: z.string(),
+        FEATURE_FLAG_SIGNALHUB_WHITELIST: z
+          .enum(["true", "false"])
+          .transform((value) => value === "true"),
```

Comment:
I think coercing to boolean should work as well:
```suggestion
        FEATURE_FLAG_SIGNALHUB_WHITELIST: z.coerce.boolean()
```

Reply:
Note — Boolean coercion with z.coerce.boolean() may not work how you expect. Any [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) value is coerced to true, and any [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value is coerced to false. That's what you can find on ZOD documentation about [coerce](https://zod.dev/?id=coercion-for-primitives)